platform/security/security-util/src/test/java/org/codice/ddf/security/util/SAMLUtilsTest.java

```diff
@@ -0,0 +1,115 @@
+
+  @Before
+  public void setUp() throws SecurityServiceException {}
+
+  @Test
+  public void testSAMLAssertionParse() throws SecurityServiceException {
+
+    SecurityToken securityToken = SAML_UTILS.getSecurityTokenFromSAMLAssertion(SAML_ASSERTION);
+    assertThat(securityToken, CoreMatchers.is(instanceOf(SecurityToken.class)));
+  }
+
+  @Test
+  public void testSAMLAssertionParseFail() throws SecurityServiceException {
+
+    SecurityToken securityToken = SAML_UTILS.getSecurityTokenFromSAMLAssertion(BAD_SAML_ASSERTION);
+    assertThat(securityToken, CoreMatchers.is(instanceOf(SecurityToken.class)));
```

Comment:
Can you assert something else about the security token? This test is performing the same assertion as the above test but this test name suggests that something should be different.

Reply:
fixed